src/Microsoft.Identity.Abstractions/DownstreamApi/IDownstreamApi.HttpMethods.tt

```diff
@@ -64,6 +73,13 @@ namespace Microsoft.Identity.Abstractions
 <# if (hasOutput){ #>
         /// <typeparam name="TOutput">Generic output type.</typeparam>
 <# } #>
+
+<# if (hasInput && framework == "net8"){ #>
+        /// <typeparam name="JsonTypeInfo<TInput>">JSON serialization metadata for TInput</typeparam>
+<# } #>
+<# if (hasOutput && framework == "net8"){ #>
+        /// <typeparam name="JsonTypeInfo<TOutput>">JSON serialization metadata for TOutput</typeparam>
+<# } #>
```

Comment:
You might want to move the content of this line at the end of the previous one to avoid generating an extra empty line in the generated code.